code/controllers/subsystem/ticker.dm

```diff
@@ -661,18 +621,6 @@ SUBSYSTEM_DEF(ticker)
 			//Break chain since this has a sleep input in it
 			addtimer(CALLBACK(player, /mob/dead/new_player.proc/make_me_an_observer), 1)
 
-/datum/controller/subsystem/ticker/proc/load_mode()
```

Comment:
Is there some persistent method for configuring dynamic next round ? Events used to be set up one round before then resetting the server. Also helps with local dev to keep it same.

Reply:
Persistent? No, but you can force the roundstart ruleset to be extended without anyone seeing.